components/Auction/createNewAuction.js

```diff
@@ -6,39 +6,70 @@ import fetchSelfDetails from '../../utils/fetchSelfDetails';
 
 const BASE_API_URL = process.env.NEXT_PUBLIC_BASE_API_URL;
 const AUCTIONS_URL = `${BASE_API_URL}/auctions`;
+const WALLET_URL = `${BASE_API_URL}/wallet`;
 
 const CreateNewAuction = () => {
   const router = useRouter();
 
-  const [itemType, setItemType] = useState();
+  const [itemType, setItemType] = useState('neelam');
```

Comment:
Using string directly can create typo error in future. Also, if there's a change in 'neelam' to some other coin, we'll end up replacing it at multiple places.
You can create a const coin = 'neelam' and use them everywhere.

Reply:
Done